go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/filecoin-project/go-address v0.0.2-0.20200218010043-eb9bb40ed5be
 	github.com/filecoin-project/go-crypto v0.0.0-20191218222705-effae4ea9f03
 	github.com/filecoin-project/go-fil-commcid v0.0.0-20200208005934-2b8bd03caca5
-	github.com/filecoin-project/specs-actors v0.2.0
+	github.com/filecoin-project/specs-actors v0.2.1-0.20200428232403-f0282340f59a
```

Comment:
Please don't do this in this PR, do it in a different one that's only about actor upgrades. 

I'm a bit torn about whether to object pointing to a non-released version at all. This increases integration costs all round. There's another release coming which lotus, gfc, and chain validation will all need to consume. Doing this cycle lots of times is a time suck.